containers/unit-tests/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -eux
+
+./autogen.sh --prefix=/usr --enable-strict --with-systemdunitdir=/tmp
```

Comment:
We've had it for a long time,  I just wonder why we use /tmp here. We should certainly let configure figure this out by itself. But that's for another cleanup PR, I made a TODO.